modules/ROOT/pages/dataweave-cookbook-perform-basic-transformation.adoc

```diff
@@ -5,11 +5,11 @@ endif::[]
 :keywords: studio, anypoint, transform, transformer, format, xml, json, dataweave, data weave, datamapper, dwl, dfl, dw, output structure, input structure
 // :page-aliases: 4.3@mule-runtime::dataweave-cookbook-perform-basic-transformation.adoc
 
-This simple DataWeave example changes the XML input to JSON output by using selector expressions without any functions. It also changes the order and names of some of the fields. Note that more complex transformations usually require the use of the `map` or `mapObject` function.
+These simple DataWeave examples change the XML input to JSON output. Note that more complex transformations usually require the use of the `map` or `mapObject` function.
 //LINK TO DW 1.0 EXAMPLES:
 include::partial$dataweave1-links.adoc[tag=dataweave1Examples]
 
-The DataWeave script maps the names (or keys) for the output fields to values of the input fields. The input fields are specified with selector expressions.
+The DataWeave script maps the names (or keys) for the output fields to values of the input fields. The input fields are specified with selector expressions without any functions. It also changes the order and names of some of the fields.
```

Comment:
The DataWeave script> The **following** DataWeave script...
It also changes > **The script** also changes...